client/src/components/User.jsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import UserExtension from "./UserExtension.jsx";
+
+export default class User extends React.Component {
+  constructor(props) {
+    super(props);
+  }
```

Comment:
Considering refactoring to a purely functional stateless component since the User component doesn't manage its own state.